### ISSUE TITLE
Documentation references 'st' without definition

### ISSUE THREAD

**Issue by generatz** [schemathesis/schemathesis#1000]
In docs at https://schemathesis.readthedocs.io/en/stable/python.html?highlight=settings#using-additional-hypothesis-strategies
**_st_**, first used here:
```
@schema.given(data=st.data())
```
is not defined.

I believe prepending this line might define it appropriately:
```
from hypothesis import strategies as st
```
If so, please add to example code, if not, please elaborate.

**Comment by Stranger6667** [comment]
Hi @generatz !

Nice catch! Sure, will fix it tomorrow and re-check other examples

**Comment by Stranger6667** [comment]
Fixed in 67382582de08d341874e5ec3ba58f8718d9a67d9 

The new release will be in a few days - for now, the `stable` docs version still contains the old code, but the `latest` one is fixed :)
